apps/portals/stopadportal/src/config/routesConfig.tsx

```diff
@@ -41,7 +41,7 @@ const routes: RouteObject[] = [
       {
         path: 'Apply/FormSubmission',
         element: (
-          <SectionLayout>
+          <SectionLayout customMaxWidth="100%">
```

Comment:
Any reason you can't just do this?
```suggestion
          <SectionLayout ContainerProps={{ maxWidth: false }}>
```

Reply:
No, that way works perfectly!